src/graph/edge.rs

```diff
@@ -103,6 +105,19 @@ impl Edge {
             direction * force_value
         }
     }
+
+    pub fn is_point_in_shape(&self, point: Vec2<f32>) -> bool {
+        let max_triangle_field =
+            Vec2::triangle_area(self.from, self.to, self.from + BASE_STROKE_WIDTH);
+
+        let triangle_area = Vec2::triangle_area(self.from, self.to, point);
+
+        if triangle_area <= max_triangle_field {
+            return true;
+        }
+
+        false
+    }
```

Comment:
Ale sprytne

Reply:
Co się dzieje jak mamy `myszkę --------------- from --------------- to` w jednej linii?

Reply:
Buguje się xd leci fix

Reply:
Wygląda, że teraz działa